webapp/src/pages/registration/Registration.js

```diff
@@ -1,11 +1,39 @@
 import React, { Component } from "react";
 import "./Registration.css";
 import RegistrationComponent from "./components/RegistrationComponent"
+import GuestRegistrationComponent from "./components/GuestRegistrationComponent"
+import { registrationService } from "../../services/registration";
+
+const DEFAULT_EVENT_ID = process.env.REACT_APP_DEFAULT_EVENT_ID || 1;
 
 export default class Registration extends Component {
-    render() {
-      return (
-       <RegistrationComponent></RegistrationComponent>
-      );
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      GuestRegistration: null
     }
-  }
\ No newline at end of file
+  }
+
+  componentDidMount() {
+    registrationService.determineIfGuest(DEFAULT_EVENT_ID).then(response => {
+      if (response !== "404") {
```

Comment:
Rather specifically check for ```response === 200```, in this case a response of 400 set would ```GuestRegistration: true```.

Reply:
Changed